src/include/gurt/common.h

```diff
@@ -211,6 +211,18 @@ extern "C" {
 		goto label;			\
 	} while (0)
 
+#define D_FPRINTF(...)							\
```

Comment:
minor nit, this does have some required arguments such as the stream and format string.